pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -987,7 +988,7 @@ func requiresTopicInValue(s Sink) bool {
 	return s.getConcreteType() == sinkTypeWebhook
 }
 
-func makeChangefeedJobDescription(
+func makeChangefeedDescription(
 	ctx context.Context,
 	changefeed *tree.CreateChangefeed,
 	sinkURI string,
```

Comment:
any reason why you didnt make the change to the comment on line 1001 like in the original pr?